labscript_devices/FunctionRunner/blacs_workers.py

```diff
@@ -14,9 +14,9 @@
 from time import monotonic
 import numpy as np
 import labscript_utils.h5_lock
+from labscript_utils.shot_utils import get_shot_globals
 import h5py
 from blacs.tab_base_classes import Worker
-import runmanager
 import runmanager.remote
```

Comment:
@chrisjbillington @philipstarkey Question: is this `runmanager.remote` import actually doing something here or can I remove it?

Maybe it is just default imported since most people use the FunctionRunner to update Runmanager globals in a feedback loop anyway?